doc/design/design_model_zoo_pai.md

```diff
@@ -0,0 +1,97 @@
+# Model Zoo on PAI TensorFlow
+
+## Introduction
+
+[Alibaba PAI](https://www.alibabacloud.com/product/machine-learning) is an end-to-end platform that provides various machine learning algorithms to meet users' data mining and analysis requirements. PAI supports TensorFlow and many other machine learning frameworks. SQLFlow model zoo also works on PAI. [Model Zoo Design Doc](./design_model_zoo.md) is a high level design for SQLFlow model zoo. This document is about how to do model training, model prediction, and model analysis using SQLFlow model zoo on PAI.
+
+## Background
+### Submit a Task to PAI
```

Comment:
I'm confusing about the relationship between PAI, PAI Tensorflow, and PAI task ... Maybe we can explain these concepts first?

Reply:
> I'm confusing about the relationship between PAI, PAI Tensorflow, and PAI task ... Maybe we can explain these concepts first?

Done.